bpo-46196: document method cmd.Cmd.columnize (#30303)
python/cpython: Doc/library/cmd.rst

   :meth:`complete_\*` method is available.  By default, it returns an empty list.

.. method:: Cmd.columnize(list, displaywidth=80)

   Method called to display a list of strings as a compact set of columns.
   Each column is only as wide as necessary.

>    Columns are separated by two spaces (one was not legible enough).

Reviewer: ```suggestion
   Columns are separated by two spaces for readability.
```
